src/containers/tokens/index.js

```diff
@@ -216,12 +216,16 @@ class Tokens extends PureComponent {
         >
           WITHDRAW PNK
         </Button>
+      </div>,
+
+      <div key={1}>
```

Comment:
Can we make the key more unique? So changes don't require reincrementing. Could just be `key="bonding-curve-totals"`

Reply:
Will change as suggested.